src/experimental/test/property/arbitraries/candid/simple_type_arbs/definition_arb.ts

```diff
@@ -65,20 +55,20 @@ export function SimpleCandidDefinitionArb(
         );
 }
 
-function generateImports(candidType: SimpleCandidType, api: Api): Set<string> {
-    if (api === 'class') {
-        if (candidType === 'Principal') {
-            return new Set(['IDL', 'Principal']);
-        }
-        return new Set(['IDL']);
+function generateImports(candidType: SimpleCandidType): Set<string> {
+    if (candidType === 'Principal') {
+        return new Set(['IDL', 'Principal']);
     }
-    return new Set([candidType]);
+    return new Set(['IDL']);
 }
 
 function toIDL(candidType: SimpleCandidType): string {
     if (candidType === 'Void') {
         return '';
     }
+    if (candidType === 'blob') {
```

Comment:
Let's discuss this

Reply:
there were already two other specific exceptions in the simple type arb (the void idl above, and the unique imports for principal) It seemed like a simple work around, with precedence, that enabled us to greatly simplify the blob arb.